src/main/java/frc/robot/subsystems/Shooter/ShooterSubsystem.java

```diff
@@ -0,0 +1,78 @@
+package frc.robot.subsystems.Shooter;
+
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.spark.SparkBase.ControlType;
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkClosedLoopController;
+import com.revrobotics.spark.SparkMax;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Configs.ShooterConfigs;
+
+/**
+ * ShooterSubsystem controls the shooter mechanism of the robot.
+ * It manages motor speed, velocity, and PID control for the shooter.
+ */
+public class ShooterSubsystem extends SubsystemBase {
+    private final SparkMax topShooterSpark = new SparkMax(2, SparkMax.MotorType.kBrushless);
+    private final RelativeEncoder topShooterEncoder = topShooterSpark.getEncoder();
+    private final SparkClosedLoopController topShooterClosedLoopController = topShooterSpark.getClosedLoopController();
+
+    private final SparkMax bottomShooterSpark = new SparkMax(1, SparkMax.MotorType.kBrushless);
+    private final RelativeEncoder bottomShooterEncoder = bottomShooterSpark.getEncoder();
+    private final SparkClosedLoopController bottomShooterClosedLoopController = bottomShooterSpark
+            .getClosedLoopController();
+
+    /**
+     * Constructs a new ShooterSubsystem and configures the shooter motors.
+     */
+    public ShooterSubsystem() {
+        // Configure motors
+        topShooterSpark.configure(ShooterConfigs.shooterConfig, ResetMode.kResetSafeParameters,
+                PersistMode.kPersistParameters);
+        bottomShooterSpark.configure(ShooterConfigs.shooterConfig.inverted(true), ResetMode.kResetSafeParameters,
+                PersistMode.kPersistParameters);
+    }
+
+    /**
+     * This method is called periodically by the scheduler.
+     * It publishes shooter velocity to the SmartDashboard.
+     */
+    @Override
+    public void periodic() {
+        SmartDashboard.putNumber("TopShooterVelocity", topShooterEncoder.getVelocity());
+        SmartDashboard.putNumber("BottomShooterVelocity", bottomShooterEncoder.getVelocity());
```

Comment:
This is where your advantagekit values should be logged.
Here would be `Shooter/Top/Measured/Velocity` and `Shooter/Bottom/Measured/Velocity`